### DISCUSSION TITLE
Allow help to be specified

### DISCUSSION THREAD

**Discussion by DavinChurch** [aplteam/Tatin#52] [Ideas]
(Posting here for discussion...)

I would like to suggest that a package's "help" capability be available via Tatin. If a new, but optional, configuration parameter called "help" were added to the system, it could be set by the developer with a URL where package help could be found. Then Tatin could use that when the package user asked for it by either supplying them the URL or simply opening their browser to it. This could be accomplished with either a `]TATIN` command or via a `##.HELP` function in the package's `code` namespace.

**Comment by e9gille** [comment]
Sounds good to me, but presumably the help is to be found in the info_url if a package has one defined?

**Reply by DavinChurch** [reply]
I was suggesting that a new, optional configuration be possible. I consider the `info_url` to be a place where full documentation and possibly source code or other resources could be found, while `help` could provide a quick summary of syntax as a reminder of how to use something. This might be similar to the user command's `-?` switch showing simple purpose and syntax.

**Comment by aplteam** [comment]
Accepted, see issue #94, though instead of "help" the more general name "documentation" will be used.
